bin/update

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+require 'pathname'
+
+gem_root = Pathname.new(__dir__).join("..")
+
+unless gem_root.join("spec/manageiq").symlink?
+  puts "== Updating manageiq sample app =="
+  system("git pull", :chdir => gem_root.join("spec/manageiq"))
+end
```

Comment:
maybe add a notification that you are not updating, because its symlinked?

```
else
  puts "== SKIP updating of manageiq sample app because its symlinked =="
end
```

I have the feeling that will save some hours of guessing multiplied by all people that use the plugins